### ISSUE TITLE
Add POJ marks, please

### ISSUE THREAD

**Issue by cathree3** [CatharsisFonts/Cormorant#55]
These fonts is so beautiful, I hope they can be used in this Taiwanese language. To-siā (Thank you).

About the POJ: https://en.wikipedia.org/wiki/Pe%CC%8Dh-%C5%8De-j%C4%AB
POJ marks: https://tauhu.tw/gua-ji-pio/

E.g
https://github.com/justfont/open-huninn-font
https://github.com/ButTaiwan/genyo-font

**Comment by CatharsisFonts** [comment]
I'll look into it. 
Do you have a list of Unicode ranges that need to be covered? Are combining marks OK or do you also need precomposed characters?

**Comment by cathree3** [comment]
need precomposed characters.
Is this list ok?
https://docs.google.com/spreadsheets/d/1RPN_4sVAYlHaMchak1wwf2dpNKRcLil7WFHkB1ZGVuE/edit#gid=1375294313

**Comment by CatharsisFonts** [comment]
I implemented the «Latin» unicode range specified by your Google Doc in the Roman cut (no stylistic alternates etc. so far). It's in the new commit. Does that work? I noticed there are no precomposed characters with the toprightdotcomb; hope it still does its job. I did add ligatures with o and O, but that might be counterproductive...?

**Comment by cathree3** [comment]
Letters and symbols have collides, need more ligatures, Thanks
<img width="618" alt="QQ20200911-031734@2x" src="https://user-images.githubusercontent.com/13507992/92789411-679ebe00-f3dd-11ea-9b53-00b99ecb6c9f.png">

Like this font: https://github.com/justfont/open-huninn-font/blob/master/font/jf-openhuninn-1.1.ttf
Huninn has many POJ ligatures
<img width="859" alt="QQ20200911-032232@2x" src="https://user-images.githubusercontent.com/13507992/92790186-222ec080-f3de-11ea-89ac-3b71661cba6d.png">

**Comment by CatharsisFonts** [comment]
Better now?

BTW, I'll still have to propagate those changes to Italic, make the smallcaps, and implement the stylistic alternates. I've got a lot of other things on my plate right now, so that's going to take a while to happen.

**Comment by cathree3** [comment]
Great. Only a little problem, They are a little close
o͘ ó͘ ò͘ ô͘ ǒ͘ ō͘ o̍͘ ő͘ ŏ͘
<img width="811" alt="QQ20200911-105913@2x" src="https://user-images.githubusercontent.com/13507992/92856231-cbe57000-f425-11ea-9f98-2d3827f24daf.png">

Could you please add the Superscript small cap ᶰ?
code: U+1DB0
<img width="472" alt="QQ20200911-115702@2x" src="https://user-images.githubusercontent.com/13507992/92856715-562dd400-f426-11ea-9a90-e631e164ef8b.png">

**Comment by CatharsisFonts** [comment]
I added the cap version of nmod and regularized hungarumlaut to fit in with the other accents (there is a dedicated loclHUN feature to adapt it to Hungarian tastes anyway).
As for the o-dot combinations: Should I just raise the macron in that particular glyph, or is there some other subtle change between the two lines? Hard to tell by eye.

**Comment by cathree3** [comment]
This version looks great
![Uploading QQ20201005-181650@2x.png…]()

**Comment by CatharsisFonts** [comment]
That link doesn't work for me.

**Comment by cathree3** [comment]
sorry, I re-uploaded, These glyphs looks great

**Comment by CatharsisFonts** [comment]
Great! I’ll propagate the coverage to the Italics and to Ysabeau. 

Just noticed the alignment of cap accents is wobbly, especially comparing breve and tilde... will have to fix.

**Comment by CatharsisFonts** [comment]
I've propagated the POJ characters and ligatures through to smallcaps and the stylistic sets. Do they work for you?

BTW, that Wikipedia page doesn't seem to include any toprightdot combining accents. I'm assuming the periodcentered that appears here and there is a fallback solution to simulate it?

**Comment by CatharsisFonts** [comment]
I just added the Italics as well. Do they work as intended?

**Comment by cathree3** [comment]
I found these problems
Not finished yet
Please give me some more time to test

Reply to you on October 10th
Thank you for your work
![2000px 畫字](https://user-images.githubusercontent.com/13507992/95535867-1dc8e800-0a1c-11eb-9bce-8e5ab334f798.png)

**Comment by CatharsisFonts** [comment]
Both of those look correct on my side:
<img width="644" alt="Screenshot 2020-10-09 at 17 29 01" src="https://user-images.githubusercontent.com/13754106/95602532-5933d880-0a55-11eb-8d77-ca0221ce7789.png">
<img width="526" alt="Screenshot 2020-10-09 at 17 30 41" src="https://user-images.githubusercontent.com/13754106/95602549-5d5ff600-0a55-11eb-8adc-012b27beb001.png">
I assume the problem is that the ligatures are not triggered and the combining accents are used as-is. I do have the requisite OpenType code in LIGA for both of these cases, though. 

Can you try again with the newest commit (fixed export problem), and otherwise send me the exact string you're using for those samples?

**Comment by cathree3** [comment]
text: ji̍t Pài-gō͘

Sorry, I tried to export fonts with Glyphys, But still not working
I test the POJ typography in Indesign.

<img width="534" alt="QQ20201010-224243@2x" src="https://user-images.githubusercontent.com/13507992/95658046-88784180-0b4a-11eb-8f72-3ddf415af9fc.png">

**Comment by CatharsisFonts** [comment]
I think I found the problem. Your test string contained /omacron/dotaboverightcomb/ rather than /o/macroncomb/dotaboverightcomb/. I added those ligatures, too. Did it work...?

And I hope the SC and Unicase fonts work too, now...

**Comment by cathree3** [comment]
I installed them all.
thank you very much, I'm sorry I didn't find the problem before, they can work normally (except for upright)
<img width="1026" alt="QQ20201011-192340@2x" src="https://user-images.githubusercontent.com/13507992/95677307-2f1a1c00-0bf7-11eb-8991-2cce5b3dbdb6.png">

![未标题-1](https://user-images.githubusercontent.com/13507992/95676927-8bc80780-0bf4-11eb-8be6-299eac5b3d4f.png)
The point of o̍͘ , compared with others, is somewhat lower (All weights)

**Comment by CatharsisFonts** [comment]
When I paste your character into Glyphs, it looks right:
<img width="540" alt="Screenshot 2020-10-11 at 16 21 07" src="https://user-images.githubusercontent.com/13754106/95681168-aabb9f00-0bde-11eb-8a41-a19c17efe031.png">

But when I look at it in Font Book, I can reproduce your error:
<img width="616" alt="Screenshot 2020-10-11 at 16 28 54" src="https://user-images.githubusercontent.com/13754106/95681207-e0f91e80-0bde-11eb-8251-62de703c8002.png">

It looks to me like the smallcaps version of the verticallineabove is used, and possibly the smallcaps version of the dotaboveright as well. This probably has to do with the support for smallcaps ligatures that I just recently added. I'll check.

**Comment by CatharsisFonts** [comment]
Huh, my latest export now seems to work in Font Book too. Not sure what I did to help. Maybe it was a caching issue after all?
<img width="204" alt="Screenshot 2020-10-11 at 17 05 08" src="https://user-images.githubusercontent.com/13754106/95682153-148a7780-0be4-11eb-84eb-390496089f1a.png">

**Comment by cathree3** [comment]
Yes, they can work.
Except Unicase and SC, their points should be higher. 
<img width="638" alt="截屏2020-10-12上午12 21 14" src="https://user-images.githubusercontent.com/13507992/95684568-2dffe380-0c25-11eb-9c5e-f6ce97e83a1b.png">
Glyphs of the Unicase are not uniform.
m m̀ ḿ m̂ m̄ m̍ m̌ m̆
n ǹ ń n̂ n̄ n̍ ň n̆ n̋
ⁿᶰ
<img width="1027" alt="截屏2020-10-12上午12 52 52" src="https://user-images.githubusercontent.com/13507992/95684571-30fad400-0c25-11eb-801f-8688048c8fff.png">

**Comment by CatharsisFonts** [comment]
Upon recommendation from the Glyphs team, I moved all those precomposed characters from LIGA to CCMP, where they feel more at home. I found another issue that was messing up some SC ligatures (decomposition happening after rather than before glyph replacement). Hopefully this finally finishes the job? Please tell me it does. 🙄

**Comment by cathree3** [comment]
Yes, Ligatures are fine, the dot is ok.

**Comment by schriftgestalt** [comment]
I don't think it needs most of the "verticallineabovecomb" ligatures. Most of them should be handled by mark positioning. And the to remove the dot in "i" for "i_verticallineabovecomb" there should be code in the ccmp already that handles this by removing the dot whenever there is a top comping in mark next to it. 
And after looking at it a bit more, I think i doesn’t need any of the precomposed glyphs.

**Comment by CatharsisFonts** [comment]
I figure the point of the precomposed ligatures is to enable the script in applications that don’t support combining accents?
